docs/HowTo/Configure/Configure-TLS.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Configure TLS for an external signer
+---
+
+# Configure TLS
+
+You can configure TLS for communication between Teku and an external signer, for example [Web3Signer].
+
+!!! info
+
+    The [Teku and Web3Signer TLS configuration tutorial] provides instructions to create the required
+    keystores and configuration.
+
+## Prerequisites
+
+**Web3Signer prerequisites**:
+
+* [Web3Signer's password-protected PKCS12 keystore and password file].
+* [Web3Signer known clients file].
+
+**Teku prerequisites**:
+
+* [Teku's password-protected PKCS12 or JKS keystore and password file].
+* [Password-protected truststore and password file].
```

Comment:
note: the truststore is also a PKCS12 or JKS format keystore. It is only storing Web3Signer's self-signed certificate or CA certificate which signs Web3Signer's certificate.